go/gas-oracle/flags/flags.go

```diff
@@ -99,7 +99,13 @@ var (
 		Name:   "significant-factor",
 		Value:  0.05,
 		Usage:  "only update when the gas price changes by more than this factor",
-		EnvVar: "GAS_PRICE_ORACLE_SIGNIFICANT_FACTOR",
+		EnvVar: "GAS_PRICE_ORACLE_L2_GAS_PRICE_SIGNIFICANT_FACTOR",
+	}
+	MaxPercentChangeL1BaseFeeFlag = cli.Float64Flag{
+		Name:   "max-percent-change-l1-base-fee",
+		Value:  0.10,
+		Usage:  "max change of the base fee between 2 updates, a float that represents a percent",
```

Comment:
does this default represent 10% or 0.1%?

Reply:
It represents 10%. It follows the same way other config in this application works

Reply:
Plz just add something to the description of the config value that reflects this